src/integ_test/java/games/strategy/engine/lobby/server/db/EmailLimitIntegrationTest.java

```diff
@@ -29,28 +29,26 @@ public static void setup() throws SQLException {
   }
 
   @Test
-  public void testAllowsMaximumLentgh() {
+  public void testAllowsMaximumLentgh() throws Exception {
```

Comment:
Why the explicit exception declaration?

Reply:
Will be addressed in the forthcoming PR I referenced below.